src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/WardrobeMouseButtons.kt

```diff
@@ -57,11 +64,41 @@ class WardrobeMouseButtons {
         if (!NotEnoughUpdates.INSTANCE.config.wardrobeKeybinds.enableWardrobeKeybinds || !NotEnoughUpdates.INSTANCE.hasSkyblockScoreboard()) return
         val gui = event.gui as? GuiChest ?: return
         if (!Utils.getOpenChestName().contains("Wardrobe")) return
+        val chestName = Utils.getOpenChestName()
+        val chestNameRegex = "Wardrobe (\\((?<current>[0-9]+)\\/(?<total>[0-9]+)\\))".toRegex()
+        val chestNameMatch = chestNameRegex.matchEntire(chestName)
+        if (chestNameMatch == null) return
+        val totalPages = chestNameMatch.groups["total"]!!.value.toInt()
+        val currentPage = chestNameMatch.groups["current"]!!.value.toInt()
 
         for (i in keybinds.indices) {
             if (KeybindHelper.isKeyDown(keybinds[i])) {
                 if (System.currentTimeMillis() - lastClick > 300) {
-                    Utils.sendLeftMouseClick(gui.inventorySlots.windowId, 36 + i)
+                    val guiChes = event.gui as GuiChest
+                    val container = guiChes.inventorySlots as ContainerChest
+                    var slotNum = 0
+                    if ((currentPage != totalPages) && KeybindHelper.isKeyDown(NotEnoughUpdates.INSTANCE.config.wardrobeKeybinds.wardrobePageNext)) {
+                        slotNum = 53
+                    } else if ((currentPage != 1) && KeybindHelper.isKeyDown(NotEnoughUpdates.INSTANCE.config.wardrobeKeybinds.wardrobePagePrevious)) {
+                        slotNum = 45
+                    } else if (KeybindHelper.isKeyDown(NotEnoughUpdates.INSTANCE.config.wardrobeKeybinds.wardrobePageUnequip)) {
+                        var notEquipped = 0
+                        for (j in 36..44) {
+                            val slotName = container.getSlot(j).getStack().getDisplayName()
+                            if (slotName.contains("Equipped")) {
+                                slotNum = j
+                            }
+                            else {
+                                notEquipped++
+                            }
+                        }
+                        if (notEquipped == 9) return
+                    } else {
+                        slotNum = (36 + i)
+                    }
+                    if (container.getSlot(slotNum).getStack().getDisplayName().isEmpty()) return
+                    if (slotNum < 36 || ((slotNum > 45) && (slotNum != 53))) return
```

Comment:
I don't think this belongs inside of the for loop?

Reply:
whoops! rewrote